### PR TITLE
NEW Athame-zsh

### PR DESCRIPTION
**Pull request by BananchickPasha**
###### Motivation for this change
There wasn't this package so i decided to add it

###### Things done
It works, but user has to create ~/.athamerc manually. 

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

**Comment by ahiaao** [comment]
I recommend rereading [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md). Specifically the sections about commit messages and meta.description.

This does not seem to respect the directory structure. Maybe move pkgs/shells/zsh/athame-zsh.nix to pkgs/shells/zsh/zsh-athame/default.nix

athame and vimbed should probably be packaged separately.

Do you actually need to override zsh? The other zsh addons do not seem to need to do this.

Are you able to reference the patch you are applying in postPatch in the actual patch stage? Also are two of those lines in postPatch just comments? If so, then those should be removed.

File: pkgs/shells/zsh/athame-zsh.nix [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{stdenv, vim, zsh, ncurses, isNeovim ? false}:
+let athame = fetchGit {
```
Comment (prusnak) [inline pkgs/shells/zsh/athame-zsh.nix]: use `fetchFromGitHub`

File: pkgs/shells/zsh/athame-zsh.nix [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+      rev = "ded4200cb3312334d99ace794c4b308cd45f6b6e";
+      url = "https://github.com/ardagnir/athame";
+    };
+    vimbed = fetchGit {
```
Comment (prusnak) [inline pkgs/shells/zsh/athame-zsh.nix]: use `fetchFromGitHub`

File: maintainers/maintainer-list.nix [hunk 3]
```diff
@@ -7670,4 +7670,9 @@
     githubId = 52650;
     name = "Marc Busqué";
   };
+  bananPasha = {
```
Comment (prusnak) [inline maintainers/maintainer-list.nix]: use `BananchickPasha` instead of `bananPasha` to match the GitHub username

**Review by prusnak — CHANGES_REQUESTED** [review]

**Pull request closed without merging**